submission-history/src/index.ts

```diff
@@ -0,0 +1,281 @@
+// Required libs
+import express from 'express';
+import mysql from 'promise-mysql';
+var fs = require('fs');
+var multer = require('multer');
+var path = require("path");
+const bodyParser = require("body-parser");
+const server = express();
+var SqlString = require('sqlstring'); // To prevent SQL injection
+
+// Use libs
+server.use(express.json());
+server.use(express.urlencoded({ extended: true })); 
+server.use(bodyParser.text()); // for req.body
+server.use(bodyParser.urlencoded({ extended: true })); // for req.body
+// Using EJS as view engine
+server.set('view engine', 'ejs');
+
+// Needed Variables
+// Gerry
+let challengeID = [];
+let challengeName = [];
+let dateSubmittedG = [];
+let didAllTestsPass = [];
+
+// Austin
+let userName = [];
+let userId = [];
+let dateSubmittedA = [];
+
+// Constructor
+async function start() {
+
+    // Create connection
```

Comment:
I've already said this in discord but I'll leave a comment here as well in case it gets lost in the chat. Can you artificially delay your service's connection to the DB? I'm still getting errors whereby the connection to the DB fails because the DB isn't ready to accept connections. This will just be a workaround until we find a better way to detect when the DB is ready.

Reply:
Added timer before connection